### ISSUE TITLE
Elaborate to other domains

### ISSUE THREAD

**Issue by amitness** [dair-ai/nlp_paper_summaries#10]
@omarsar Thanks for starting this initiative.

I was wondering if we can expand this from `nlp_paper_summaries` to `paper_summaries` instead and group papers by the domain: "NLP", "Computer Vision" etc. There is no existing platform that curates such paper summaries in one place and so `paper_summaries` could fill that gap. Just a thought.

**Comment by omarsar** [comment]
@amitness We are considering this, any suggestions on how we can better structure the repo if we were to go with `paper_summaries`? My initial thought was to create folders and include a README in each one? So we would have separate folders for `compuater_vision`, `nlp`, etc for now? I was even considering having separate repos but could take a bit more effort to maintain, I like the concept of just having one repo. 

In addition, we would need some help (domain expertise) to maintain the different domains, create the proper groupings, etc.

**Comment by amitness** [comment]
@omarsar Separate folders with dedicated README.md in each sounds like a good idea. We could also link them from the main README.md.

**Comment by vrdn-23** [comment]
I agree with @omarsar on this! One repo sounds ideal but the main obstacle would be having the required domain expertise in order to find, accumulate and segregate papers from various domains

**Comment by omarsar** [comment]
Note to self: Look into this issue a bit closer and come up with general structure.
